civitai-downloader-v2/src/core/security/scanner.py

```
@@ -0,0 +1,744 @@
+                # Read and analyze pickle stream without executing
+                pickletools_available = True
+                try:
+                    import pickletools
+                except ImportError:
+                    pickletools_available = False
+                
+                if pickletools_available:
+                    # Analyze pickle opcodes
+                    f.seek(0)
+                    try:
+                        opcodes = []
```

Ruff (0.12.2)

Local variable `opcodes` is assigned to but never used

Remove assignment to unused variable `opcodes`

(F841)